Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -513,6 +515,31 @@ int CLuaAccountDefs::RemoveAccount(lua_State* luaVM)
     return 1;
 }
 
+int CLuaAccountDefs::SetAccountSerial(lua_State* luaVM)
+{
+    SString strName;
+    SString strSerial;
```

Comment:
Avoid using Hungarian notation in new code

Reply:
Thanks for the feedback, I will avoid using future contributions <3